src/tactic/suggest.lean

```diff
@@ -64,6 +64,8 @@ meta def allowed_head_symbols : expr → list name
 
 -- And then the generic cases:
 | (expr.pi _ _ _ t) := allowed_head_symbols t
+| `(_ ≠ _) := [`false]
+| `(¬ _ = _) := [`ne]
```

Comment:
Is this here after `pi` on purpose? If not, maybe it could go at the end of the "customisations" section along with a comment that these are special cases to help it look for ne and false lemmas.

Reply:
Not necessarily after `pi`, but it needs to be before `app`

Reply:
I'm going to make the change in a few minutes 👍🏼